src/main/java/zeenea/connector/common/ItemInventory.java

```diff
@@ -144,9 +154,17 @@ public Builder itemIdentifier(IdentificationProperty itemIdentifier) {
      *
      * @param labels the list of labels to add
      * @return the builder instance
+     * @deprecated since 2.3.0, use labelIdentifier instead
      */
+    @Deprecated(since = "2.3.0", forRemoval = true)
```

Comment:
Je t'invite à reprendre le format des notices de dépréciation utilisé précédemment : 
```suggestion
  @Deprecated(
      since =
          "Deprecated since version 2.3.0, use labelIdentifier instead. Scheduled for removal in version 3.0.0.",
      forRemoval = true)
```

Reply:
Corrigé